Storage/src/Connection/ServiceDefinition/storage-v1.json

```diff
@@ -2406,35 +2764,42 @@
     }
    }
   },
-  "notifications": {
+  "managedFolders": {
    "methods": {
     "delete": {
-     "id": "storage.notifications.delete",
-     "path": "b/{bucket}/notificationConfigs/{notification}",
+     "id": "storage.managedFolders.delete",
```

Comment:
Does this seem like breaking?

Reply:
The difference shown in git seems strange visually. Managed folders service is added. It replaces nothing.

The notifications line has been moved to ~3000 line number [here](https://github.com/googleapis/google-cloud-php/blob/0c30923879aa92642ae9f9af9780c3f6a2863b5f/Storage/src/Connection/ServiceDefinition/storage-v1.json#L3062)

Reply:
Thanks for clarifying.. I attempted to run the tests and notification tests also worked fine..